tests/testthat/test-5transformCounts.R

```diff
@@ -512,6 +512,52 @@ test_that("transformAssay", {
             all(div[pair, ] == ref)
         })
         all_correct |> all() |> expect_true()
+
+		
+		############################## INVNORM #################################
+        tse <- GlobalPatterns
+        
+        # Manual IRNT for comparison
+        invnorm_one <- function(v, offset = 0.5, ties.method = "average") {
+            ok <- !is.na(v)
+            n  <- sum(ok)
+            out <- rep(NA_real_, length(v))
+            if( n > 0L ){
+                r <- rank(v[ok], ties.method = ties.method)
+                p <- (r - offset) / (n + 1 - 2 * offset)
+                p[p <= 0] <- .Machine$double.eps
+                p[p >= 1] <- 1 - .Machine$double.eps
+                out[ok] <- qnorm(p)
+            }
+            out
+        }
```

Comment:
Is this the same function as in R file? Not sure if it gives much information to check against this same function.